rasa/core/actions/forms.py

```diff
@@ -515,8 +516,8 @@ async def _validate_if_required(
         """
         # no active_loop means that it is called during activation
         need_validation = not tracker.active_loop or (
-            tracker.latest_action_name == ACTION_LISTEN_NAME
-            and tracker.active_loop.get("validate", True)
+            tracker.latest_action.get(ACTION_NAME) == ACTION_LISTEN_NAME
```

Comment:
shorter and better for the encapsulation of `tracker`
```suggestion
            tracker.latest_action_name == ACTION_LISTEN_NAME
```